### ISSUE TITLE
Homing

### ISSUE THREAD

**Issue by BerndNeuhauser** [bullestock/4xidraw#23]
I want the plotter to go to its home position after each print. is thera a way to do this? 

So is there an option to put the grbl command at end of each internally generated gcode?

**Comment by misan** [comment]
You can add a 
;pen up +
G0 X0 Y0
at the end.

**Comment by BerndNeuhauser** [comment]
> 
> 
> You can add a
> ;pen up +
> G0 X0 Y0
> at the end.

yes, i understand but ->where<- should i write your commands? There is no "gcode file" - i work with the GUI from the extension :(

**Comment by misan** [comment]
You can add a line after https://github.com/bullestock/4xidraw/blob/master/inkscape%20driver/grbl_serial.py#L122

        self.write('G0 X0 Y0\r') # also within the if above
